app/models/talk.rb

```diff
@@ -45,7 +46,8 @@ class Talk < ActiveRecord::Base
 
   validates :speaker_twitter,
     format: { with: /\A@.+\z/ },
-    allow_nil: true
+    allow_nil: true,
+    unless: :skip_twitter_validations
```

Comment:
This works, but a bit too much. We are going to skip validation on every talk creation, hence allowing incorrectly formatted twitter nicknames. I don't think we want to skip validation, but rather allow blank values, which is possible using the correct option